README.md

```diff
@@ -63,7 +63,11 @@ To get started with Solace Agent Mesh, follow these steps:
 
    _(Use `-eb` to combine build and run steps.)_
 
-5. **Send a Request**:
+5. **Connect to the Web Interface**:
+
+   Open the web interface at [http://localhost:5050](http://localhost:5001) or with the port specified during `init`.
```

Comment:
link text and hyperlink don't match both should use port 5001

Reply:
Fixed!